core/tests/ts-integration/src/env.ts

```diff
@@ -25,22 +25,19 @@ export async function waitForServer() {
     const l2Provider = new zksync.Provider(l2NodeUrl);
 
     reporter.startAction('Connecting to server');
-    let ready = false;
     for (let i = 0; i < maxAttempts; ++i) {
         try {
             await l2Provider.getNetwork(); // Will throw if the server is not ready yet.
-            ready = true;
             reporter.finishAction();
+            // TODO (PLA-842): [workaround] sleep for 10s.
+            await zksync.utils.sleep(10 * 1000);
```

Comment:
Nit: Such sleeps are often forgotten, so maybe it makes sense to do

```typescript
reporter.warn("TODO (PLA-842): Connected to server, waiting 10 seconds for it to warm up");
```

this may incentivize people to prioritize the corresponding ticket 😅

Reply:
done